Gems/Atom/RHI/Code/Source/RHI/Object.cpp

```diff
@@ -8,11 +8,18 @@
 #include <Atom/RHI/Object.h>
 #include <AzCore/Module/Environment.h>
 #include <AzCore/std/parallel/atomic.h>
+#include <AzCore/std/parallel/scoped_lock.h>
 
 namespace AZ::RHI
 {
+    //TODO see comment in Object.h
+#if RHI_OBJECT_STATIC_MUTEX
+    AZStd::recursive_mutex Object::s_nameMutex;
+#endif
```

Comment:
This is not used anywhere

Reply:
Yes, it is used. For developers who want to try STATIC vs per-object Mutex.